tasks/config.yml

```diff
@@ -35,6 +35,10 @@
     force: true
   notify: restart Solr
 
+- name: Solr | Gather OS specific vars
+  include_vars: "{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
+  when: ansible_facts['distribution'] == "Debian" and ansible_facts['distribution_major_version'] == "11"
```

Comment:
A bit of personal preference: I think this may be too tied to the case of Debian 11. I would rather generate a vars file to each single distro we support so it can be easily read.

Reply:
Yes, I also think this would be cleaner, made a commit :)